src/views/teachers/landing/landing.jsx

```diff
@@ -22,107 +22,255 @@ const Landing = () => (
+                            allowTransparency="true"
                             frameBorder="0"
-                            src="https://www.youtube.com/embed/uPSuG063jhA?border=0&wmode=transparent"
+                            height="180"
+                            scrolling="no"
+                            src={`https://fast.wistia.net/embed/iframe/h8hay3nnt4?seo=false&videoFoam=true`}
+                            title=""
+                            width="320"
                         />
                     </div>
                 </FlexRow>
             </div>
             <div className="band">
                 <SubNavigation className="inner">
-                    <a href="#in-practice">
+                    <a href="#resources">
                         <li>
-                            <FormattedMessage id="teacherlanding.inPracticeTitle" />
+                            <FormattedMessage id="teacherlanding.resourcesAnchor" />
                         </li>
                     </a>
-                    <a href="#resources">
+                    <a href="#connect">
                         <li>
-                            <FormattedMessage id="teacherlanding.resourcesAnchor" />
+                            <FormattedMessage id="teacherlanding.connectAnchor" />
+                        </li>
+                    </a>
+                    <a href="#news">
+                        <li>
+                            <FormattedMessage id="teacherlanding.newsAnchor" />
                         </li>
                     </a>
                     <a href="#teacher-accounts">
                         <li>
-                            <FormattedMessage id="general.teacherAccounts" />
+                            <FormattedMessage id="teacherlanding.teacherAccountsAnchor" />
                         </li>
                     </a>
                 </SubNavigation>
             </div>
         </TitleBanner>
 
         <div className="inner">
-            <section id="in-practice">
-                <span className="nav-spacer" />
-                <h2>
-                    <FormattedMessage id="teacherlanding.inPracticeTitle" />
-                </h2>
-                <p className="intro">
-                    <FormattedMessage id="teacherlanding.inPracticeIntro" />
-                </p>
-                <FlexRow className="general-usage">
-                    <p><FormattedHTMLMessage id="teacherlanding.generalUsageSettings" /></p>
-                    <p><FormattedHTMLMessage id="teacherlanding.generalUsageGradeLevels" /></p>
-                    <p><FormattedHTMLMessage id="teacherlanding.generalUsageSubjectAreas" /></p>
-                </FlexRow>
-            </section>
             <section id="resources">
                 <span className="nav-spacer" />
-                <h2><FormattedMessage id="general.resourcesTitle" /></h2>
+                <h2><FormattedMessage id="teacherlanding.educatorResources" /></h2>
```

Comment:
If this is some sort of title, I think it would help to include "title" or "heading" or something like that in the name